config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -91,7 +91,7 @@ presubmits:
         - --runtime-config=batch/v2alpha1=true
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-storage-snapshot
         - --test_args=--ginkgo.focus=\[Feature:VolumeSnapshotDataSource\] --ginkgo.skip=\[Disruptive\]|\[Flaky\] --minStartupPods=8
-        - --timeout=80m
+        - --timeout=120m
```

Comment:
Can you also bump the top level job timeout to 150 to match the config on the serial job: https://github.com/kubernetes/test-infra/blob/3686b3f215bf471f93a5ba1bd33c0b78e7b5b9a6/config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml#L185

Reply:
And ci-kubernetes-e2e-snapshot below should also be updated